app/github_public_repo/estimate_csv_creator.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+import pandas as pd
+from flask import current_app as app
+
+from app.database_etl.airtable_records_handler import get_all_records, airtable_get_request
+
+# Load env variables for Airtable request
+AIRTABLE_API_KEY = app.config['AIRTABLE_API_KEY']
+AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL = app.config['AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL']
+
+# Get columns that should be pulled into CSV
+headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+data = airtable_get_request(AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL, headers)
+
+# Get csv columns and create csv
+try:
+    # Extract fields
+    records = data['records']
+    fields = [x['fields']['Field Name'] for x in records]
+    logging.info("Successfully retrieved field names from Airtable")
+
+    # Get estimates from Rapid Review: Estimates table for specified fields
+    csv_records = get_all_records(fields)
+
+    # Convert to df, reorder cols alphabetically and as save as csv
```

Comment:
Can we do better than reordering cols alphabetically here? There are certain columns which index the data - country, sampling dates, etc. Seems like we ought to arrange them in this order.

We can arrange them in AirTable in this fashion, and then order them in this CSV in the same fashion.

I know it's programatically equivalent -- but oftentimes, when working with data like this, I'll manually look at the CSV to see how it's laid out. and that's much easier if it's laid out in an order that makes sense to me